src/test/java/org/folio/rest/impl/AutomatedPatronBlocksAPITest.java

```diff
@@ -428,29 +452,34 @@ public void blockWhenMaxOutstandingFeeFineBalanceLimitIsExceededAllLimitsExist()
 
   @Test
   public void allLimitsAreExceeded() {
+    String loanId = randomId();
     createLimitsForAllConditions();
 
-    BigDecimal outstandingBalance = BigDecimal.valueOf(
-      LIMIT_VALUES.get(MAX_OUTSTANDING_FEE_FINE_BALANCE) + 5);
-
     int maxOverdueRecallLimit = LIMIT_VALUES.get(RECALL_OVERDUE_BY_MAX_NUMBER_OF_DAYS);
     DateTime dueDate = now().minusDays(maxOverdueRecallLimit + 1);
 
     OpenLoan overdueRecalledLoan = new OpenLoan()
-      .withLoanId(randomId())
+      .withLoanId(loanId)
       .withRecall(true)
+      .withItemLost(true)
       .withDueDate(dueDate.toDate());
 
-    int numberOfOpenLoans = Math.max(
-      LIMIT_VALUES.get(MAX_NUMBER_OF_ITEMS_CHARGED_OUT), Math.max(
-        LIMIT_VALUES.get(MAX_NUMBER_OF_OVERDUE_ITEMS),
-        LIMIT_VALUES.get(MAX_NUMBER_OF_OVERDUE_RECALLS))) + 1;
+    int numberOfOpenLoans = Collections.max(Arrays.asList(
+      LIMIT_VALUES.get(MAX_NUMBER_OF_LOST_ITEMS),
+      LIMIT_VALUES.get(MAX_NUMBER_OF_ITEMS_CHARGED_OUT),
+      LIMIT_VALUES.get(MAX_NUMBER_OF_OVERDUE_ITEMS),
+      LIMIT_VALUES.get(MAX_NUMBER_OF_OVERDUE_RECALLS))) + 1;
 
     List<OpenLoan> openLoans = fillListOfSize(overdueRecalledLoan, numberOfOpenLoans);
 
-    int numberOfLostItems = LIMIT_VALUES.get(MAX_NUMBER_OF_LOST_ITEMS) + 1;
+    List<OpenFeeFine> openFeeFines = singletonList(
+      new OpenFeeFine()
+        .withBalance(BigDecimal.valueOf(LIMIT_VALUES.get(MAX_OUTSTANDING_FEE_FINE_BALANCE) + 1))
+        .withLoanId(loanId)
+        .withFeeFineId(randomId())
+        .withFeeFineTypeId(randomId()));
```

Comment:
Does that make sense to move OpenFeeFine creation to separate method with parameters? I have seen it at least twice

Reply:
Done, thanks.